x-pack/test/api_integration/apis/ml/annotations/get_annotations.ts

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import _ from 'lodash';
```

Comment:
You're only using `omit` from lodash, so I'd suggest to just import this one:
```
import { omit } from 'lodash';
```

Reply:
Updated in https://github.com/elastic/kibana/pull/73068/commits/f45a447e78d38637d6649fcb92e6d3235b13c906